src/proc.h

```diff
@@ -316,6 +319,9 @@ class job_iterator_t {
         return job;
     }
 
+    /// Returns a hash of all jobs currently running that can be used to determine if jobs have
+    /// changed. Does not reflect changes to the states of these jobs.
+    size_t hash() const;
```

Comment:
Where is this used?

Reply:
No, sorry that entire commit should be dropped. I used to use a snapshot of the jobs to determine if we can safely elide waitpid(2) calls, but that logic has been changed now.